workers/unity/Assets/Playground/Scripts/Cubes/CubeMovementSystem.cs

```diff
@@ -19,20 +20,25 @@ public struct Data
 
         [Inject] private Data data;
 
-        private static Vector3 speed = new Vector3(2, 0, 0);
-
-        private Vector3 origin;
-
-        protected override void OnCreateManager(int capacity)
+        public struct WorkerData
         {
-            base.OnCreateManager(capacity);
-
-            var worker = WorkerRegistry.GetWorkerForWorld(World);
-            origin = worker.Origin;
+            public readonly int Length;
+            [ReadOnly] public SharedComponentDataArray<WorkerConfig> WorkerConfigs;
         }
 
+        [Inject] private WorkerData workerData;
+
+        private static Vector3 speed = new Vector3(2, 0, 0);
+
         protected override void OnUpdate()
         {
+            if (workerData.Length == 0)
+            {
+                new LoggingDispatcher().HandleLog(LogType.Error, new LogEvent("This system should not have been run without a worker entity"));
```

Comment:
Can this use the default dispatcher?

Reply:
discussed offline: we don't have the worker here, so we need to create a logging dispatcher

Reply:
This kind of error message isn't very useful to a user. Can be explicit about what the error is but we should probably also add something which suggests that it's because they might have started the worker incorrectly.